lib/main.js

```diff
@@ -90,6 +93,19 @@ function logServerPort (address, port) {
   logger.info(logMessage);
 }
 
+function fixArgvForParser () {
+  // this function mutates argv to ensure that argparse gets the 'server'
+  // subcommand in the case where the user provides no subcommand
+  if (process.argv.length < 2) {
+    process.argv.push('server');
+    return;
+  }
+
+  if (!_.includes(['driver', 'server', '-h'], process.argv[2])) {
+    process.argv.splice(2, 0, 'server');
```

Comment:
is it necessary to mutate process.argv ?

Reply:
yes, argparse reads global process.argv. We could re-set it after running the parser to be on the safe side though.

Reply:
I found a different solution that involves overriding the parseArgs method. Yay for no mutation.